### ISSUE TITLE
TypeError: JSONEditor is not a constructor when using Jest

### ISSUE THREAD

**Issue by amitlevy21** [json-editor/json-editor#518]
Hey,
I am having build issues when test with jest.

### General information
The build passes and when I test visually in the browser JSONEditor works, the form is generated as I expect. 

However, when I test with jest. I get the following error:
`TypeError: JSONEditor is not a constructor`

I have seen #127 but it did not help.

This is my Vue component
```vue
<template>
  <div id='editorHolder'></div>
</template>

<script>
import * as JSONEditor from '@json-editor/json-editor'

export default {
  name: 'JsonEditContentDefault',
  data: function () {
    return {
      resultContent: null,
      schema: {},
      editor: {}
    }
  },
  methods: {
    initialJsonToFormData () {
      return {}
    },
    initialJsonToRawText () {
      return this.$store.state.initialJsonString
    },
    formDataToJson () {
      return {}
    }
  },
  watch: {
    resultContent () {
      let resultJson = this.resultContent
      this.$store.commit('setResultJson', {resultJson})
    }
  },
  mounted () {
    const el = document.getElementById('editorHolder')
    this.schema = JSON.parse(this.$store.state.engineSchema)
    this.resultContent = JSON.parse(this.initialJsonToRawText())
    this.editor = new JSONEditor(el, {
      schema: this.schema,
      theme: 'bootstrap4',
      compact: true,
      disable_properties: true,
      disable_array_reorder: true
    })
    this.editor.setValue(this.resultContent)

    this.editor.on('change', () => {
      this.resultContent = this.editor.getValue()
    })
  }
}
</script>

<style>
  #editorHolder h3 {
    font-size: 1rem;
  }
  #editorHolder .btn-group {
    display: inline-block;
  }
</style>

```

This is my jest file containing a simple unit test that was passing before I started to use JSONEditor
```javascript
  it('initialJsonToRawText - calculate raw text as full file', () => {
    const wrapper = shallowMount(JsonEditContentDefault, {store, localVue})

    let initialJsonString = JSON.stringify(getTestInitialJSON())
    store.commit('setInitialJsonString', {initialJsonString})

    expect(wrapper.vm.initialJsonToRawText())
      .toEqual(JSON.stringify(getTestInitialJSON()))
  })
```
- json-editor version: 1.3.5

### Expected behavior
The test should pass as it passed before using JSONEditor.
### Actual behavior
Build error: 
```
console.error node_modules/vue/dist/vue.runtime.common.js:589
    [Vue warn]: Error in config.errorHandler: "TypeError: JSONEditor is not a constructor"

  console.error node_modules/vue/dist/vue.runtime.common.js:1739
    TypeError: JSONEditor is not a constructor
        at VueComponent.mounted
```
### Steps to reproduce the behavior
1. Create a new Vue 2 project
2. Create a component like the one I'm using above.
3. Create the most simple unit test using jest
4. Running the test would show the build error

Thank you

**Comment by pmk65** [comment]
Try the develop/2.x branch, which is Webpack based.

**Comment by amitlevy21** [comment]
@pmk65 now I get an error when viewing via the browser and nothing is rendered. I'm getting the same error as in #519

**Comment by sirockin** [comment]
Sorry @amitlevy21 there is currently a separate issue on develop/2.x which I'm trying to fix: #519

**Comment by sirockin** [comment]
Hi @amitlevy21. 

I have fixed in #521 (currently only on `develop/2.x` branch, but you now need to import the library as follows:
```
import { JSONEditor } from '@json-editor/json-editor/dist/jsoneditor.amd.js
```
I'll aim to make this path nicer before we merge onto master - see #522
